dora/core/server/common/src/main/java/alluxio/underfs/AbstractUfsManager.java

```diff
@@ -225,6 +227,22 @@ public void removeMount(long mountId) {
     mMountIdToUfsInfoMap.remove(mountId);
   }
 
+  /**
+   * Gets an instance for the given UFS URI and configuration, if such exists.
+   *
+   * @param ufsUri the URI of the UFS
+   * @param ufsConf the configuration for the UFS
+   * @return a UFS instance, or none if not registered
+   */
+  public Optional<UnderFileSystem> get(AlluxioURI ufsUri, UnderFileSystemConfiguration ufsConf) {
```

Comment:
`get()` and `getOrAdd()` are semantically too indirect, I can't tell what the methods do based on the names. What about `UfsResolver.resolveUfsForPath(path)`?

Reply:
> I can't tell what the methods do based on the names. 

`get` returns a `Optional<UnderFileSystem>` while `getOrAdd` returns `UnderFileSystem`. The difference should be obvious enough.